CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_amplitude_source v0.2.1
+## 🎉 Feature Updates 🎉: 
```

Comment:
We should highlight that these are breaking changes and feature updates.

Reply:
done